python/lsst/ctrl/mpexec/cmdLineFwk.py

```diff
@@ -797,3 +813,86 @@ def _importGraphFixup(self, args: SimpleNamespace) -> Optional[ExecutionGraphFix
                 raise ValueError("Graph fixup is not an instance of ExecutionGraphFixup class")
             return fixup
         return None
+
+    def preExecInitQBB(self, task_factory: TaskFactory, args: SimpleNamespace) -> None:
+
+        # Load quantum graph. We do not really need individual Quanta here,
+        # but we need datastore records for initInputs, and those are only
+        # available from Quanta, so load the whole thing.
```

Comment:
Probably not worth doing it now, but I think this is another argument for making PreExecInit a special Quantum in the future.

Reply:
👍 I want to avoid changing persistent QGraph format on this branch, I feel like it's not quite trivial and deserves a dedicated ticket.